usage_examples/tests/test_readme_examples.py

```diff
@@ -54,8 +55,8 @@ def test_example_2_ner_hf():
         factory = "llm"
 
         [components.llm.task]
-        @llm_tasks = "spacy.NER.v2"
-        labels = ["PERSON", "ORGANISATION", "LOCATION"]
+        @llm_tasks = "spacy.TextCat.v2"
+        labels = ["COMPLIMENT", "INSULT"]
 
```

Comment:
This could be an additional test, but I don't see why we're removing the old one for `spacy.NER.v2`? As we discussed that we wouldn't actually be deprecating these old versions, let's keep their tests intact as well.

Reply:
Tests for old versions of NER and SpanCat are in `tests/legacy` (not sure whether that's the ideal location, but good enough for now).